shared/featureconfig/flags.go

```diff
@@ -75,17 +75,15 @@ var (
 		Usage: "Cache filtered block tree by maintaining it rather than continually recalculating on the fly, " +
 			"this is used for fork choice.",
 	}
-	disableProtectProposerFlag = &cli.BoolFlag{
-		Name: "disable-protect-proposer",
-		Usage: "Disables functionality to prevent the validator client from signing and " +
+	enableProtectProposerFlag = &cli.BoolFlag{
+		Name: "enable-protect-proposer",
```

Comment:
Please deprecate these instead of deleting them - we can no longer delete flags from the repo